### DISCUSSION TITLE
Can logs be cent over the internet for centralized logging?

### DISCUSSION THREAD

**Discussion by damienhull** [Security-Onion-Solutions/securityonion#2505] [Unsupported Versions] [answered]
I'm new to security onion. Going through the free training. Everything looks awesome. Thought popped into my head. 
1. We're all working from home
2. How do you send logs back to a central location with a mobile workforce? 

**Here's what I would like to have**
1. SecurityOnion in the cloud
2. Laptop with windows 10
3. logs sent over the internet to SecurityOnion
4. Is this possible? 

Thanks!

**Comment by weslambert** [comment, marked as answer]
Depending on how you wanted to achieve this, you should be able to do this by pointing logs to Security Onion via Osquery, Wazuh, Winlogbeat using TLS, etc.  If you have a large number of endpoints, you could even have a dedicated Osquery/Fleet node to assist with this (or use WEF for WLB -- dedicated Wazuh node option coming soon) .

**Reply by SanibelJack** [reply]
Hi Wes,
I too would like to do this as well and have tried. I get an error ( error: desc = \"transport: authentication handshake failed: x509: certificate is valid for yyy.yyy.yyy.yyy, not xxx.xxx.xxx.xxx\""). where yyy.yyy.yyy is the private IP of my manager / search server and xxx.xxx.xxx is the external/public ip of my manager / search server.
my Win10 machine that is at a location different then my manager search servers. Any suggestions?

I've adjusted the filebeat.yml on the remote Win10 pc to reflect the public IP and have written forwarding rules on myfirewall to forward to my internal manager / server.

As a side note logstash works fine in this configuration.


Thanks

**Reply by damienhull** [reply]
> Depending on how you wanted to achieve this, you should be able to do this by pointing logs to Security Onion via Osquery, Wazuh, Winlogbeat using TLS, etc. If you have a large number of endpoints, you could even have a dedicated Osquery/Fleet node to assist with this (or use WEF for WLB -- dedicated Wazuh node option coming soon) .

Sorry for the late response. I haven't had a chance to look into this. Just knowing that we can do this over TLS is awesome. Thanks!
